File: CS50x/4. Memory/6.c
/*

> Date Created: July 26, 2023
> Author: Ishaan Rastogi
> Purpose: To compare strings
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working

*/

#include"cs50.h"
#include<stdio.h>
#include<string.h>

int main()
{
    string s = get_string("s: ");
    string t = get_string("t: ");

    if ( strcmp(s,t) == 0)
    {
        printf("Same\n");
    }
    else
    {
        printf("Different\n");
    }
}

/* 

Ctrl + Shift + `

Terminal Code Loading...

gcc 6.c -o 6 cs50.c
./6

*/

File: Web Development Projects/15 Intermediate JS/6.js
/*
> Date Created: June 10, 2024
> Author: Ishaan Rastogi
> Purpose: Playing with arrays
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working
*/

var guestList = [ "Ishaan", "Anil Kapoor", "Tisca Chopra",, "Ashish Vidyarthi", "Neil Bhoopalam", "Sapna Pabbi" ]; 
console.log(guestList); // Output - Whole list
console.log(guestList.length); // Output - 6
guestList[0]; // Output - Ishaan


//Checks if your name is there or not
var guestName = prompt ("What is your name?");
guestList.includes(guestName);

/* 
For Terminal Code... Ctrl + Shift + `
*/